app/test/frontend/handlers/landing_test.dart

```diff
@@ -53,23 +51,6 @@ void main() {
       );
     });
 
-    testWithProfile('/ without a working search service', fn: () async {
```

Comment:
Is this removing the fallback search entirely?

Reply:
No, we are keeping that. This test became obsolete somewhere in the past 2-3 months, where we've updated the fallback mechanism and also did the isolate-based search migration. I think this kind of mocking is not that useful in tests, as it does hide structural changes, and now, months after the actual change, it is harder to trace it back.